src/catchup/ApplyBufferedLedgersWork.cpp

```diff
@@ -68,8 +68,9 @@ ApplyBufferedLedgersWork::onRun()
 
     auto applyLedger = std::make_shared<ApplyLedgerWork>(mApp, lcd);
 
-    auto predicate = [&]() {
```

Comment:
I like that we don't need to capture everything by reference anymore (much safer this way), thanks for cleaning this up!

Reply:
Yes, this was a good improvement. Thanks for suggesting it!